### PR TITLE
Feat: add <Table.Cell> component

### PR DESCRIPTION
**Pull request by juanesquintero**
- Add `<Table.Cell>` component
- Enable `<Table>` inner border preview 
- Add `<Table>` margin support

### CODE REVIEW

**Comment by juanesquintero** [comment]
@wyozi Let me know what I'm missing to make the `<Table>` node open to support `Cell`, but not as child and keep the `rows` as prop

**Comment by juanesquintero** [comment]
Still missing styles details on the render for download

**Comment by juanesquintero** [comment]
@wyozi  Let me know when you reviewed please 😁

File: src/normalizer.ts [hunk 1]
```diff
@@ -298,7 +307,30 @@ export const normalizeText = (t: TextChild): InternalTextPart[] => {
...
+        : [],
+    style: {
+      ...style,
+      ...normalizedCoordinates,
+      color: style.color ? normalizeHexColor(style.color) : null,
+      fontFace: style.fontFace ?? DEFAULT_FONT_FACE,
+      fontSize: style.fontSize ?? DEFAULT_FONT_SIZE,
+    },
+    colSpan: node.props.colSpan,
+    rowSpan: node.props.rowSpan,
```
Comment (wyozi) [inline src/normalizer.ts:328]: Should these be here since this is a generic function for text nodes?

File: pages/index.tsx [hunk 2]
```diff
@@ -170,6 +170,15 @@ const code = `ReactPPTX.render(
...
+          [<Table.Cell 
+            colSpan={2} 
+            style={{ 
+              align: "center", 
+              backgroundColor: '#115599', 
+              color: 'white'
+            }}>
+            Title
+          </Table.Cell>],
           ["foo", <Text style={{ align: "right", backgroundColor: '#404040' }}>bar</Text>],
```
Comment (wyozi) [inline pages/index.tsx:182]: I guess all `<Text>` elements used in `rows` could be made into `<Table.Cell>` now to be a bit more consistent?

**Comment by wyozi** [comment]
Looks very good 👌 Just left a few notes

**Comment by juanesquintero** [comment]
Thanks! Will make those for tomorrow 😁

File: pages/index.tsx (hunk 2 above)
Comment (juanesquintero) [reply pages/index.tsx]: Done ✅

File: src/normalizer.ts (hunk 1 above)
Comment (juanesquintero) [reply src/normalizer.ts]: Done ✅

**Comment by wyozi** [comment]
LGTM! Thanks

**Pull request merged**